pkg/es/writer/esBulkHandler.go

```diff
@@ -239,6 +288,21 @@ func HandleBulkBody(postBody []byte, ctx *fasthttp.RequestCtx, rid uint64, myid
 			items[inCount-1] = resp_status_201
 		}
 	}
+
+	pleBatches := utils.ConvertSliceToMap(allPLEs, func(ple *writer.ParsedLogEvent) string {
+		return ple.GetIndexName()
+	})
+
+	for indexName, plesInBatch := range pleBatches {
+		err = ProcessIndexRequestPle(tsNow, indexName, uint64(len(line)),
```

Comment:
we need to remove the `len(line)` param from this func and figure out the `len(line)` inside the function for each individual PLE.